server.go

```diff
@@ -186,6 +235,248 @@ func main() {
 	}
 }
 
+const msgUpstreamParticipantCreateError = "could not create upstream participant. response status: %s"
+
+func upstreamNewParticipant(c echo.Context, p participant) (id string, err error) {
+	item := leaderboard_item{}
+	item.UserName = p.GitHubName
+	item.Slug = p.GitHubName
+	item.Score = 0
+	item.Archived = false
+	item.Draft = false
+
+	payload := leaderboard_payload{}
+	payload.Fields = item
+
+	body, err := json.Marshal(payload)
+	if err != nil {
+		return
+	}
+
+	req, err := http.NewRequest("POST", fmt.Sprintf("%s/collections/%s/items?live=true", webflowBaseAPI, webflowCollection), bytes.NewReader(body))
```

Comment:
If your URL's start to get a bit long, and you want to use something more "checkable": https://play.golang.org/p/KbVcoEB1F-N

That should be an example that creates something similar to what you've got here!

(not a strong suggestion, just informational)

Reply:
Good to know. Going to leave as is for now. Lift rug, sweep. ;)